docs/quickstart.rst

```diff
@@ -22,16 +22,13 @@ In a Terminal, use :code:`pip` to install the package from `PyPI`_.
 
 If you intend to use the :class:`~improved_user.factories.UserFactory`
 provided by the package to allow for testing with |factory_boy|_, you
-can specify so during install.
+can either install |factory_boy| separately, or use the modified installation
```

Comment:
This RST directive needs a `_` at the end of it (like the one on the previous line).

Once you [build the docs](http://django-improved-user.readthedocs.io/en/latest/contributing.html#your-first-documentation-contribution), hover over items that should be linked to verify they work as desired. Sphinx does not provide errors for items like this, unfortunately.

Reply:
I agree. I think it's reasonable to assume that the standard user for this project is more likely to want the factory inclusion than not. I'll amend this to reflect that.